scripts/hostcfgd

```diff
@@ -394,8 +398,9 @@ class FeatureHandler(object):
 
             for suffix in reversed(feature_suffixes):
                 cmds.append("sudo systemctl stop {}.{}".format(feature_name, suffix))
-                cmds.append("sudo systemctl disable {}.{}".format(feature_name, feature_suffixes[-1]))
-                cmds.append("sudo systemctl mask {}.{}".format(feature_name, feature_suffixes[-1]))
+            
+            cmds.append("sudo systemctl disable {}.{}".format(feature_name, feature_suffixes[-1]))
+            cmds.append("sudo systemctl mask {}.{}".format(feature_name, feature_suffixes[-1]))
```

Comment:
@wenyiz2021 I think the initial logic to disable a feature is to `stop`, `disable` the feature, then `mask` the correspoding systemd configuration file, right?

Reply:
Are these 2 lines supposed to be in the for loop? If so, then you have indentation issue. Please set your edit to expand tabs to white spaces and use white spaces only.

Reply:
My comments was a question, not a request. I was asking if your intended to put the 2 lines in the loop. Your answer is no. I think we are good.

Reply:
thanks. Sorry for the late reply, I forgot to send my review comments and just noticed it yday.